src/js/components/HostTable.js

```diff
@@ -5,6 +5,7 @@ import {StoreMixin} from 'mesosphere-shared-reactjs';
 import {Tooltip} from 'reactjs-components';
 
 var HostTableHeaderLabels = require('../constants/HostTableHeaderLabels');
+import IconServers from '../../img/components/icons/icon-medium/servers.svg?name=IconServers';
```

Comment:
Hahaha, understandably! We opted to go this route (as opposed to creating simple React components that return SVG markup) to allow the design team to easily maintain the icons going forward.